testjenkins.sh

```diff
@@ -125,13 +125,14 @@ run_ctest () {
 
 run_pytest_normal () {
 	run enable_environment
-	python -m pytest -m "not equinor_test" --durations=10
+	python -m pytest -m "not equinor_test" --durations=10 tests
```

Comment:
Why do you specify `tests` now?

Reply:
Because else it will do discovery on the entire test-data folder (including the Equinor folder), which is both very slow and also causes crash since some files dont have read permission.

Reply:
The only question left is why it worked before?:)

Reply:
Because the time it takes to scan the test-data folder when there is only the local folder is very fast. So it's fine that pytest scans the folder and finds no tests, since there is no sideeffects to this :)